src/Aspire.Hosting/Dcp/DcpHostService.cs

```diff
@@ -73,7 +73,17 @@ public async Task StopAsync(CancellationToken cancellationToken = default)
         _shutdownCts.Cancel();
         if (_logProcessorTask is { } task)
         {
-            await task.ConfigureAwait(ConfigureAwaitOptions.SuppressThrowing);
+            try
+            {
+                await task.ConfigureAwait(false);
+            }
+            catch (OperationCanceledException)
+            {
+            }
+            catch (Exception ex)
+            {
+                Console.Error.WriteLine($"Error in logging socket processor: {ex}");
```

Comment:
ILogger would be better than console unless there is a reason for console here.

Reply:
I am following the pattern used elsewhere in the file, but I see now that current usage is split between Console & ILogger